projects/openNeuroClient/openNeuroClient.py

```diff
@@ -11,46 +12,76 @@
 # import project modules from rt-cloud
 from rtCommon.utils import loadConfigFile, stringPartialFormat
 from rtCommon.clientInterface import ClientInterface
+from rtCommon.bidsArchive import BidsArchive
 
 # path for default configuration toml file
-defaultConfig = os.path.join(currPath, 'conf/openNeuro.toml')
+defaultConfig = os.path.join(currPath, 'conf/openNeuroClient.toml')
```

Comment:
I think this would be more cross-platform compatible if it were `'conf', 'openNeuroClient.toml'` so `os.path.join` can choose the proper separator.